commands/publish.go

```diff
@@ -305,7 +306,8 @@ func PrintPublishUsage() {
 	util.PrintUtil("  -%s\t\tForce Major version bump of 'jobVersion' in manifest on disk if publish conflict found\n",
 		constants.JobVersionMajor)
 
-	util.PrintUtil("\nExample: \tseed publish -in example-0.1.3-seed:0.1.3 -r hub.docker.com -o geoint -jm -P\n")
```

Comment:
I'm slightly torn on the removal of organization from examples, especially if we are using Docker Hub as an example. Hub will always be used with an organization, since the global namespace is reserved for the official images.

Reply:
True, docker hub requires org (aka user).  I guess the idea was do we want to encourage/discourage orgs in our private registry on the highside?